client/src/pages/Listings/useStyles.ts

```diff
@@ -0,0 +1,5 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({}));
```

Comment:
Here and below, it's a good practice to create these `useStyles` files only when necessary. Sometimes you end up not using them :)

Reply:
Applied